android-pdf-viewer/src/main/java/com/github/barteksc/pdfviewer/listener/Callbacks.kt

```diff
@@ -127,6 +137,22 @@ class Callbacks {
         }
     }
 
+    fun setOnDraw(onDrawListener: OnDrawListener?) {
+        this.onDrawListener = onDrawListener
+    }
+
+    fun getOnDraw(): OnDrawListener? {
+        return onDrawListener
+    }
+
+    fun setOnDrawAll(onDrawAllListener: OnDrawListener?) {
+        this.onDrawAllListener = onDrawAllListener
+    }
+
+    fun getOnDrawAll(): OnDrawListener? {
+        return onDrawAllListener
+    }
+
```

Comment:
Same as above just make the properties public

Reply:
Done